frontend/src/components/EditProgramModal.js

```diff
@@ -253,6 +253,34 @@ function ProgramDuration({
   setLocked,
   disabled = false,
 }) {
+  const timeStep = useSelector((state) => state.settings.settings.timeStep);
+  const defaultDurations = {
+    [15 * 60 * 1000]: [
+      ["0:15", "15 min"],
+      ["0:30", "30 min"],
+      ["0:45", "45 min"],
+      ["1:00", "1 hod"],
+      ["1:30", "1,5 hod"],
+      ["2:00", "2 hod"],
+    ],
+    [10 * 60 * 1000]: [
```

Comment:
@marekdedic any suggestions for default durations for 10 min time increments?

Reply:
How about 10, 30, 40, 60, 90 and 120? I think 40 is used more often than 20... ;)